include/config.h

```diff
@@ -8,7 +8,7 @@ typedef struct {
   double dt;
   int m; // Size in X.
   int n; // Size in Y.
-  double r;
+  int seed; // Random generator seed
```

Comment:
Please add a dot _(`.`)_ at the end of the comments. :+1:

Reply:
Done!